router/main_test.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+	"time"
+)
+
+func Test_makeHandler(t *testing.T) {
+	config := []struct {
+		Title              string
+		Host               string
+		RequestURI         string
+		ExpectedSuffix     string
+		UpstreamURL        string
+		Timeout            time.Duration
+		ExpectedStatusCode int
+	}{
+		{
+			Title:              "Right configuration with status 500 unrelated to function",
+			Host:               "martindekov.example.xyz/",
+			RequestURI:         "myfunction",
+			ExpectedSuffix:     "martindekov-myfunction",
```

Comment:
What about the whole expected upstream URL?

Reply:
Well the response body is not as expected 
```
Proxying to: http://localhost:8080/function/martindekov-myfunction
... ```
we only print.. that but rather
```
unable to scale function [martindekov-myfunction], err: error scaling HTTP code 500, http://faas-swarm:8080/system/scale-function/martindekov-myfunction
```
because we resent the request but I can find a way to send `Proxying to: ...` message in the body of the response